src/main/java/org/cloudfoundry/promregator/endpoint/EndpointConstants.java

```diff
@@ -1,11 +1,11 @@
 package org.cloudfoundry.promregator.endpoint;
 
 public interface EndpointConstants {
-	static final String ENDPOINT_PATH_DISCOVERY = "/discovery";
-	static final String ENDPOINT_PATH_SINGLE_ENDPOINT_SCRAPING = "/metrics";
-	static final String ENDPOINT_PATH_SINGLE_TARGET_SCRAPING = "/singleTargetMetrics";
-	static final String ENDPOINT_PATH_PROMREGATOR_METRICS = "/promregatorMetrics";
-	static final String ENDPOINT_PATH_CACHE_INVALIDATION = "/cache/invalidate";
+	String ENDPOINT_PATH_DISCOVERY = "/discovery";
```

Comment:
I still don't get it that attributes of interfaces in Java are implicitly `final`, but be it...

Reply:
I found this handy description for interfaces

All the members (methods and fields) of an interface are public.
All the methods in an interface are public and abstract (except static and default).
All the fields of an interface are public, static and, final by default.